### PR TITLE
Azure Theme 8.0: Tag Button variant for DefaultButton and PrimaryButton

### PR DESCRIPTION
**Pull request by Jacqueline-ms**
## Previous Behavior

No previous behavior - new feature

## New Behavior

These designs deviate from the portal only in the dark theme text color. The black text against the blue background was not readable. I also noticed that the rest background remains the same for al states in dark theme, but the border color changes. I believe this PR will unblock the issue of no tags in Azure Theme for now, and we can fine tune the background colors / text / border colors at a later time.

Light & High Contrast Light themes - rest, hover, active, focus
image
image

Dark & High Contrast Dark themes - rest, hover, active, focus
image

## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/5835524f6f0ed2e463e8cae0b5ae29252b6bfbc4'>5835524f6f0ed2e463e8cae0b5ae29252b6bfbc4</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/azure-themes/src/azure/AzureColors.ts [hunk 1]
```diff
@@ -2,9 +2,12 @@ import { IAzureSemanticColors } from './IAzureSemanticColors';
```
Comment (fabricteam) [inline packages/azure-themes/src/azure/AzureColors.ts:2]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27186&commitId=fbbd295961ec6a53322ce424b428ecdab4501ebb&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1045 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Review by robtaft-ms — APPROVED** [review]
Thank you  Jackie!!!!

File: change/@fluentui-azure-themes-d76d2c27-2e5d-4c90-bb94-2d9e9ebb0ede.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Azure Theme Tag Button variant for primary and default buttons",
```
Comment (khmakoto) [inline change/@fluentui-azure-themes-d76d2c27-2e5d-4c90-bb94-2d9e9ebb0ede.json:3]:
```suggestion
  "comment": "feat: Adding Azure Theme Tag Button variant for primary and default buttons",
```

**Review by khmakoto — APPROVED** [review]

File: change/@fluentui-azure-themes-d76d2c27-2e5d-4c90-bb94-2d9e9ebb0ede.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-azure-themes-d76d2c27-2e5d-4c90-bb94-2d9e9ebb0ede.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27186&commitId=cb3ff0dfa6dcd1874205ecbf6af55740b562ef89&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1045 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

File: change/@fluentui-azure-themes-d76d2c27-2e5d-4c90-bb94-2d9e9ebb0ede.json (hunk 3 above)
Comment (fabricteam) [inline change/@fluentui-azure-themes-d76d2c27-2e5d-4c90-bb94-2d9e9ebb0ede.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27186&commitId=7ad11a53c3829eaaaa9b895383e34bb6b67c2710&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1045 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Pull request merged**